packages/react-wildcat-handoff/src/utils/serverRender.js

```diff
@@ -2,7 +2,7 @@
 const ReactDOM = require("react-dom/server");
 const Router = require("react-router");
 const serverContext = require("./serverContext.js");
-const Helmet = require("react-helmet"); // eslint-disable-line import/no-unresolved
+const {Helmet} = require("react-helmet"); // eslint-disable-line import/no-unresolved
```

Comment:
Do we still need the eslint-disable-line here? I think we might be able to remove it now